java/mykola-melnyk/src/com/zoolatech/lecture7/tasks/_2/Calc_v_1.java

```diff
@@ -0,0 +1,19 @@
+package com.zoolatech.lecture7.tasks._2;
+
+public class Calc_v_1 implements Calc {
+    private volatile int storedValue;
```

Comment:
you don't need to use `volatile` if the access to a variable is protected with `synchronized`